dashboard/test/controllers/api/v1/pd/teacher_progress_report_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class Api::V1::Pd::TeacherProgressReportControllerTest < ::ActionController::TestCase
+  test 'admins can view the report' do
+    sign_in create(:admin)
+    get :index
+    assert_response :success
+  end
+
+  test 'workshop organizers can view the report' do
+    sign_in create(:workshop_organizer)
+    get :index
+    assert_response :success
+  end
```

Comment:
Tests can be combined here as well.

In fact, I'm wondering how many of these report tests can be combined. It seems like there are some tests that are viewable by admins and workshop organizers only - you can probably write this as one test with a map.